taxonium_component/src/Deck.jsx

```diff
@@ -384,7 +378,59 @@ function Deck({
             settings={settings}
           />
         </View>
-        <View id="main"></View>
+
+        <View id="map">
+          <div
+            id="map-div"
+            key={crypto.randomUUID()}
```

Comment:
Is this unique key each time important? (I assume so but just checking)

Reply:
Usually I err on the side of including an `id` so that React can more efficiently calculate rerenders. I believe this specific case was added because a warning in the console, so it seems like React was struggling to update `Views`.

Reply:
I think you're totally right here, so I just swapped out the `crypto.uuid` with the `"map-div-key"` string. Thanks!